### PR TITLE
more conformance

### PR DESCRIPTION
**Pull request by missinglink**
more conformance, mostly tests

- when backend responds with query errors (from ES) then pass them to the user as errors, show geojson

### CODE REVIEW

**Comment by dianashk** [comment]
:+1:

File: controller/place.js [hunk 1]
```diff
@@ -22,11 +22,15 @@ function setup( backend ){
     });
 
     service.mget( backend, query, function( err, docs ) {
-      // error handler
-      if( err ){ return next( err ); }
 
+      // error handler
+      if( err ){
+        req.errors.push( err.message ? err.message : err );
```
Comment (orangejulius) [inline controller/place.js]: More transparent error handling = excellent!

File: sanitiser/_geo_autocomplete.js [hunk 2]
```diff
@@ -8,8 +8,7 @@ module.exports = function sanitize( raw, clean ){
   var messages = { errors: [], warnings: [] };
 
   try {
-    geo_common.sanitize_coord( 'focus.point.lat', clean, raw['focus.point.lat'], LAT_LON_IS_REQUIRED );
-    geo_common.sanitize_coord( 'focus.point.lon', clean, raw['focus.point.lon'], LAT_LON_IS_REQUIRED );
+    geo_common.sanitize_point( 'focus.point', clean, raw, LAT_LON_IS_REQUIRED );
```
Comment (orangejulius) [inline sanitiser/_geo_autocomplete.js]: sanitisze_point! hooray

**Comment by orangejulius** [comment]
based on the number of added tests I expect us to never have a bug again. :ship:

**Pull request merged**